smart_commits.egg-info/PKG-INFO

```diff
@@ -0,0 +1,13 @@
+Metadata-Version: 1.1
```

Comment:
.egg-info folder/files are unnecessary. I don't think we need them to create pip installable.